pkg/controller/controller.go

```diff
@@ -590,6 +667,32 @@ func (p *csiProvisioner) Provision(options controller.VolumeOptions) (*v1.Persis
 	return pv, nil
 }
 
+func stripPrefixedParameters(param map[string]string) (map[string]string, error) {
```

Comment:
nit: `remove` instead of `strip`? At first I was under the impression that the prefix is being stripped off of the parameter key.